src/api-reference/request/v4.get-started.markdown

```diff
@@ -33,7 +32,7 @@ The Request V4 API exposes four different resources:
 
 Resource|Description
 ---|---
-Request|You can read, create, update or delete a Request and get the list of Requests.
+Request|You can read, create, update or delete a Request and get the list of an existing Request.
```

Comment:
Reformulate

Reply:
Not for the current PR, we'll discuss in more detail for a future documentation update.